recipes/bioconductor-monocle/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = 'monocle' %}
+{% set version = '2.4.0' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url:
+    - https://bioconductor.org/packages/release/bioc/src/contrib/{{ name }}_{{ version }}.tar.gz
+  md5: fef48c57d5da92fae71cb501d8603cf0
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+
+requirements:
+  build:
+    - r-base
+    - r-biobase
```

Comment:
Because there not all of them are needed, you need to store the packages, rebuild them ... maintain them. The skeleton generator works reliable, but not 100%.

Yet, to some degree we try this - it's just a lack of man-power as usual.